stable/redis-ha/templates/NOTES.txt

```diff
@@ -1,24 +1,23 @@
-Redis cluster can be accessed via port 6379 on the following DNS name from within your cluster:
+Redis cluster can be accessed via port {{ .Values.redis.port }} on the following DNS name from within your cluster:
 {{ template "redis-ha.fullname" . }}.{{ .Release.Namespace }}.svc.cluster.local
 
 To connect to your Redis server:
 
 {{- if .Values.auth }}
 1. Get the randomly generated redis password:
-   
    echo $(kubectl get secret {{ template "redis-ha.fullname" . }} -o "jsonpath={.data['auth']}" | base64 -D)
 
-2. Connect to the Redis master pod that you can use as a client:
+2. Connect to the Redis master pod that you can use as a client. By default the {{ template "redis-ha.fullname" . }}-server-0 pod is configured as the master:
 
-   kubectl exec -it $(kubectl get pod -o jsonpath='{range .items[*]}{.metadata.name} {.status.containerStatuses[0].state}{"\n"}{end}' -l redis-role=master | grep running | awk '{print $1}') bash
+   kubectl exec -it {{ template "redis-ha.fullname" . }}-server-0 bash -n {{ .Release.Namespace }}
 
 3. Connect using the Redis CLI (inside container):
 
    redis-cli -a <REDIS-PASS-FROM-SECRET>
 {{- else }}
 1. Run a Redis pod that you can use as a client:
 
-   kubectl exec -n {{ .Release.Namespace }} -it $(kubectl get pod -n {{ .Release.Namespace }} -o jsonpath='{range .items[*]}{.metadata.name} {.status.containerStatuses[0].state}{"\n"}{end}' -l redis-role=master | grep running | awk '{print $1}') bash
+   kubectl exec -it {{ template "redis-ha.fullname" . }}-server-0 bash -n {{ .Release.Namespace }}
 
 2. Connect using the Redis CLI:
```

Comment:
The redis server host should be changed, since service name changed.

Reply:
this command is correct for the new version, works on my test environment: 
`kubectl exec -it test-redis-ha-server-0 bash -n default`

Reply:
@prodriguezdefino If I understand correctly, he's referring to the `2. Connect using..` section. This section slipped by me and I'll update it accordingly. Thanks @ceshihao !